planetiler-core/src/main/java/com/onthegomap/planetiler/util/CommonStringEncoder.java

```diff
@@ -1,41 +1,58 @@
 package com.onthegomap.planetiler.util;
 
+import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
-import java.util.concurrent.ConcurrentMap;
 import java.util.concurrent.atomic.AtomicInteger;
 import javax.annotation.concurrent.ThreadSafe;
 
 /**
- * A utility for compressing up to 250 commonly-used strings (i.e. layer name, tag attributes) into a single byte.
+ * A utility for compressing commonly-used strings (i.e. layer name, tag attributes).
  */
 @ThreadSafe
 public class CommonStringEncoder {
```

Comment:
Let's not have this class store two separate encodings internally - we construct a separate instance for layers and attributes anyway.  Can we just make this one support large values, but maybe add a nested class for bytes, something like:

```java
public static class AsByte {
  private final CommonStringEncoder = new CommonStringEncoder();
  public String decode(byte id) { ... }
  public byte encode(String string) { range check then encode()... }
}
```

?